### PR TITLE
[SDK-3709] Update README

### PR DESCRIPTION
**Pull request by poovamraj**
### Changes

- Introduced EXAMPLES as seperate file
- Expo quickstart and sample gets separate called out
- README redesign implementation

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+- [Authentication API](#authentication-api)
```
Comment (frederikprijck) [inline EXAMPLES.md:1]:
```suggestion
# Examples using react-native-auth0

- [Authentication API](#authentication-api)
```

File: EXAMPLES.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+### Get user information using user's access_token
+
+```js
+auth0.auth
+  .userInfo({token: 'the user access_token'})
+  .then(console.log)
+  .catch(console.error);
+```
+
+This endpoint requires an Access Token that was granted the `/userinfo` audience. Check that the authentication request that returned the Access Token included an audience value of `https://{YOUR_AUTH0_DOMAIN}.auth0.com/userinfo`.
```
Comment (frederikprijck) [inline EXAMPLES.md:53]:
```suggestion
This endpoint requires an access token that was granted the `/userinfo` audience. Check that the authentication request that returned the access token included an audience value of `https://{YOUR_AUTH0_DOMAIN}.auth0.com/userinfo`.
```

File: EXAMPLES.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+Using Organizations, you can:
+
+- Represent teams, business customers, partner companies, or any logical grouping of users that should have different ways of accessing your applications, as organizations.
+- Manage their membership in a variety of ways, including user invitation.
+- Configure branded, federated login flows for each organization.
+- Implement role-based access control, such that users can have different roles when authenticating in the context of different organizations.
+- Build administration capabilities into your products, using Organizations APIs, so that those businesses can manage their own organizations.
+
+Note that Organizations is currently only available to customers on our Enterprise and Startup subscription plans.
```
Comment (frederikprijck) [inline EXAMPLES.md:181]:
```suggestion
[Organizations](https://auth0.com/docs/organizations) is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.

Note that Organizations is currently only available to customers on our Enterprise and Startup subscription plans.
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,39 +1,31 @@
...
-
 [![Build Status][circleci-image]][circleci-url]
 [![NPM version][npm-image]][npm-url]
 [![Coverage][codecov-image]][codecov-url]
 [![License][license-image]][license-url]
 [![Downloads][downloads-image]][downloads-url]
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Freact-native-auth0.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Freact-native-auth0?ref=badge_shield)
 
-## Important Notices
+📚 [Documentation](#documentation) • 🚀 [Getting Started](#getting-started) • ⏭️ [Next Steps](#next-steps) • 💬 [Feedback](#feedback)
```
Comment (frederikprijck) [inline README.md:10]: Do these emoji's work when published to npm or are should we use the below suggestion instead?

```suggestion
:books: [Documentation](#documentation) • :rocket: [Getting started](#getting-started) • :next_track_button: [Next steps](#next-steps) • :speech_balloon: [Feedback](#feedback)
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: EXAMPLES.md (hunk 3 above)
Comment (frederikprijck) [reply EXAMPLES.md:181]: As mentioned on slack, we are getting rid of some of the organizations marketing text, as we are already point to the docs above.

File: EXAMPLES.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+[Organizations](https://auth0.com/docs/organizations) is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.
+
+Using Organizations, you can:
+
+- Represent teams, business customers, partner companies, or any logical grouping of users that should have different ways of accessing your applications, as organizations.
+- Manage their membership in a variety of ways, including user invitation.
+- Configure branded, federated login flows for each organization.
+- Implement role-based access control, such that users can have different roles when authenticating in the context of different organizations.
+- Build administration capabilities into your products, using Organizations APIs, so that those businesses can manage their own organizations.
+
```
Comment (frederikprijck) [inline EXAMPLES.md:180]:
```suggestion

```

As mentioned on slack, we are getting rid of some of the organizations marketing text, as we are already point to the docs above.

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: EXAMPLES.md (hunk 5 above)
Comment (poovamraj) [reply EXAMPLES.md:180]: I read that we are removing it from README @frederikprijck, that's why I added it to EXAMPLES.md. You think we should remove it here as well?

File: README.md [hunk 6]
```diff
@@ -410,290 +402,36 @@ try {
...
+---
 
-This project is licensed under the MIT license. See the [LICENSE](LICENSE) file for more info.
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="./assets/auth0_brand_light.png"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/auth0_brand_dark.png" width="150">
+    <img alt="Auth0 Logo" src="./assets/auth0_brand_light.png" width="150">
+  </picture>
+</p>
```
Comment (frederikprijck) [inline README.md:431]:
```suggestion
<p align="center">
  <picture>
    <source media="(prefers-color-scheme: light)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png"   width="150">
    <source media="(prefers-color-scheme: dark)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_dark_mode.png" width="150">
    <img alt="Auth0 Logo" src="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png" width="150">
  </picture>
</p>
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]
We can use the CDN links for the footer logo, and drop the logo's from the repository.

File: EXAMPLES.md (hunk 5 above)
Comment (frederikprijck) [reply EXAMPLES.md:182]: I think that's the intention, yes.

File: README.md (hunk 4 above)
Comment (poovamraj) [reply README.md]: The emoji are Unicode encoded so it should be the only format that will work in the most cases. For example, using the shortforms doesn't work in VS Code

Screenshot 2022-10-20 at 12 21 12

I am checking whether NPM supports this format.

File: README.md [hunk 7]
```diff
@@ -410,290 +402,36 @@ try {
...
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_dark_mode.png" width="150">
+    <img alt="Auth0 Logo" src="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png" width="150">
+  </picture>
+</p>
+<p align="center">Auth0 is an easy to implement, adaptable authentication and authorization platform. To learn more checkout <a href="https://auth0.com/why-auth0">Why Auth0?</a></p>
+<p align="center">
+This project is licensed under the MIT license. See the <a href="https://github.com/auth0/react-native-auth0/blob/master/LICENSE.md"> LICENSE</a> file for more info.</p>
```
Comment (frederikprijck) [inline README.md:434]: The license file doesn't have any extension
```suggestion
This project is licensed under the MIT license. See the <a href="https://github.com/auth0/react-native-auth0/blob/master/LICENSE"> LICENSE</a> file for more info.</p>
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**